server/models/Group.js

```diff
@@ -382,8 +382,12 @@ module.exports = function(Sequelize, DataTypes) {
       },
 
       getRelatedGroups(limit) {
```

Comment:
For design sake, I'd rather have `getRelatedGroups(tags, limit)`
E.g.: the homepage could be considered has a super collective, and we want to show the related groups for "open source" and the ones for "meetup".